src/wiring/saveEntireWiring.ts

```diff
@@ -26,34 +29,92 @@ export default async function saveEntireWiring(
       throw e;
     }
   }
+  const connecterPath = join(path, "Connectors");
+  try {
+    await mkdir(connecterPath);
+  } catch (e: any) {
+    if (e.code !== "EEXIST") {
+      throw e;
+    }
+  }
 
   for (let i = 0; i < toc.length; i++) {
     const doc = toc[i];
     const sanitizedTitle = doc.Title.replace(/\//g, "-");
 
-    const newTitle = join(wiringPath, `${sanitizedTitle}.png`);
-
-    if (doc.Type === "Page") {
-      console.log(
-        `Downloading wiring manual page ${doc.Title} as image (#${doc.Number})`
-      );
-
-      await fetchPage(
+    // 150 = Connector views, different format handled later
+    // Always 150?
+    if (doc.Type === "Page" && doc.Number !== "150") {
```

Comment:
Is it always 150, or is it 150 only for your car-- could you see if another car has its connectors as 150? My guess, although I very well could be wrong, is that the page numbers change per car.

Reply:
It seems they are the same over a few vehicles.. switchin to use Title, should be better.